app/controllers/application_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Chitter < Sinatra::Base
+
+  set :views, proc { File.join(root, "../views") }
+  set :public_folder, proc { File.join(root, "../public") }
+
+  enable :sessions
+  set :session_secret, 'super secret'
+
+  register Sinatra::Flash
+  use Rack::MethodOverride
+
+  helpers Helpers
+
+  get '/' do
+    erb:index
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.